### PR TITLE
[sqlcipher] Bump to 4.5.3.

### PR DESCRIPTION
**Pull request by SvenPStarFinanz**
- [X] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [X] SHA512s are updated for each updated download
- [X] The "supports" clause reflects platforms that may be fixed by this new version
- [X] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [X] Any patches that are no longer applied are deleted from the port's directory.
- [X] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [X] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by SvenPStarFinanz** [comment]
The commit message was wrong, meant [sqlcipher].

File: ports/sqlcipher/portfile.cmake [hunk 1]
```diff
@@ -3,8 +3,8 @@ vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO sqlcipher/sqlcipher
-    REF v4.5.2
-    SHA512 1de5b219392bb976631857e32b4523258fd660fedb558d478e536b7e10c711c72c7e7c9062e45bd8a5ceaecbc1fee717935d2357f6811c3ddf76702167f4601b
+    REF v4.5.3
```
Comment (FrankXie05) [inline ports/sqlcipher/portfile.cmake:6]:
```suggestion
    REF "v${VERSION}"

```
We can use the new method. :)

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

File: ports/sqlcipher/portfile.cmake (hunk 1 above)
Comment (FrankXie05) [reply ports/sqlcipher/portfile.cmake:6]: It's not necessary, but I'll wait a day for the user to respond.

File: ports/sqlcipher/portfile.cmake [hunk 2]
```diff
@@ -3,8 +3,9 @@ vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO sqlcipher/sqlcipher
-    REF v4.5.2
-    SHA512 1de5b219392bb976631857e32b4523258fd660fedb558d478e536b7e10c711c72c7e7c9062e45bd8a5ceaecbc1fee717935d2357f6811c3ddf76702167f4601b
+    REF "v${VERSION}"
+
+    SHA512 27dcc83c8088ea32a2bbc3d5493d070e29976dc54ccf91d849df67be19cb553761a32b3613e293a07f50128aeb808d54f363ad12a58a47f06a61a1cc4e1eb0cd
```
Comment (FrankXie05) [inline ports/sqlcipher/portfile.cmake:8]:
```suggestion
    REF "v${VERSION}"
    SHA512 27dcc83c8088ea32a2bbc3d5493d070e29976dc54ccf91d849df67be19cb553761a32b3613e293a07f50128aeb808d54f363ad12a58a47f06a61a1cc4e1eb0cd

```
No blank lines required.

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

**Review by FrankXie05 — DISMISSED** [review]

**Review by FrankXie05 — APPROVED** [review]

**Pull request merged**